### PR TITLE
add(tools): Tailwind Components' cheatsheet

### PR DESCRIPTION
**Pull request by khatabwedaa**
| Name                 | Link                 |
| -------------------- | -------------------- |
| _Tailwindcomponents cheatsheet_ | _https://tailwindcomponents.com/cheatsheet_ |

_Tailwind CSS cheat sheet helps you to discover Tailwind CSS utilities_

---

- [x] My item is in the right category
- [x] My item is logically grouped below similar items
- [x] My item's name and description respects the conventions of the list
- [x] My item is awesome
- [x] I have read and followed the [contribution guidelines](https://github.com/aniftyco/awesome-tailwindcss/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Review by innocenzi — CHANGES_REQUESTED** [review]
The description doesn't respect the guidelines.

**Comment by innocenzi** [comment]
@joshmanders I just realized the name of this project doesn't respect the usage allowed by Tailwind Labs (https://tailwindcss.com/brand) - do we have to delete each item from the list with this domain name? 

@khatabwedaa I noticed you have a brand, Highscore Studio, in the copyright of your project's site. Shouldn't you use that in place of `tailwindcomponents.com`?

**Comment by joshmanders** [comment]
> do we have to delete each item from the list with this domain name?

That's a good question, let me confer with Adam.

**Comment by khatabwedaa** [comment]
@innocenzi what I have to change?

File: README.md [hunk 1]
```diff
@@ -217,6 +217,7 @@
 - 🌍 [Tailwind Cheat Sheet](http://nerdcave.com/tailwind-cheat-sheet) - Tailwind CSS class names cheat sheet.
 - 🌍 [Tailwind Cheat Sheet](https://github.com/LeCoupa/awesome-cheatsheets/blob/master/frontend/tailwind.css) - Tailwind CSS class names in one single file.
 - 🌍 [Tailwind Cheat Sheet](https://umeshmk.github.io/Tailwindcss-cheatsheet) - Tailwind CSS class names, variants and directives cheat sheet.
+- 🌍 [Tailwindcomponents Cheatsheet](https://tailwindcomponents.com/cheatsheet) - Tailwind CSS cheat sheet helps you to discover Tailwind CSS utilities.
```
Comment (innocenzi) [inline README.md:220]:
```suggestion
- 🌍 [Tailwind Cheat Sheet](https://tailwindcomponents.com/cheatsheet) - Tailwind CSS class names in a searchable page.
```

**Review by innocenzi — APPROVED** [review]

**Comment by innocenzi** [comment]
Changed it for you

**Comment by khatabwedaa** [comment]
Thanks 👍

**Pull request merged**